csrankings.csv

```diff
@@ -2411,6 +2411,7 @@ Boguslaw Obara,Durham University,https://www.dur.ac.uk/computer.science/staff/pr
 Bohyung Han,Seoul National University,http://cvlab.snu.ac.kr/~bhhan,9aaeCToAAAAJ
 Boi Faltings,EPFL,http://liawww.epfl.ch/~faltings,X21nbgcAAAAJ
 Bojan Cukic,UNC - Charlotte,https://cci.uncc.edu/directory/bojan-cukic,B7N812UAAAAJ
+Bolei Zhou,Chinese University of Hong Kong,http://bzhou.ie.cuhk.edu.hk/,9D4aG8AAAAAJ

```

Comment:
Not in a CS department. Please verify inclusion criteria. (I note that you have not checked a single checkbox in the contributions checklist.)